### ISSUE TITLE
Add .travis.yml

### ISSUE THREAD

**Issue by ericelliott** [ericelliott/universal-react-boilerplate#30]
See [prod-module-boilerplate](https://github.com/ericelliott/prod-module-boilerplate/blob/master/.travis.yml)

**Comment by nkbt** [comment]
You could also add `--silent` to `npm run check`  to avoid npm's not very helpful error messages. Other errors will be shown fine.

Example of such npm error
```
npm ERR! isomorphic-express-boilerplate@1.2.3 lint: `eslint app`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the isomorphic-express-boilerplate@1.2.3 lint script.
npm ERR! This is most likely a problem with the isomorphic-express-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint app
npm ERR! You can get their info via:
npm ERR!     npm owner ls isomorphic-express-boilerplate
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 14.3.0
npm ERR! command "/Users/nkbt/.nvm/v0.10.38/bin/node" "/Users/nkbt/.nvm/v0.10.38/bin/npm" "run" "lint"
npm ERR! cwd /Users/nkbt/nkbt/universal-react-boilerplate
npm ERR! node -v v0.10.38
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/nkbt/nkbt/universal-react-boilerplate/npm-debug.log
npm ERR! not ok code 0
```

**Comment by ericelliott** [comment]
:+1:

**Comment by nkbt** [comment]
fixed with #32
